manual/managing_github.md

```diff
@@ -0,0 +1,52 @@
+# Numfocus Github permission process
+
+Numfocus defines default permission levels for repositories. The intention is to share as much as we can publicly and restrict access only when required. There are two types of repositories with access restriction. The ones that contains NumFocus board and management sensitive information, the other ones contains data that can’t be made public for other reasons (surveys, tools, etc.).
+
+Access to sensitive repositories must be requested to the board. Access should be reviewed on a quarterly? basis.
```

Comment:
quarterly is frequent. probably only do that if it's one persons responsibility. broader review could be 1x/year